docs/.vuepress/components/TextFieldDoc.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<StackLayout class="textField-wrapper">
+		<StackLayout orientation="horizontal">
+			<Label text="Textfield:" />
+			<TextField 
+			id="TextField"
+			:text="text"
+			:hint="hint"
+			:editable="editable"
+			:maxLength="maxLengthNumeric"
+			:secure="secure"
+			:keyboardType="keyboardType"
+			:returnKeyType="returnKeyType"
+			:autocorrect="autocorrect"
+			@input="text = $event"
+			@textChange="eventOutput = 'textChange'"
+			@returnPress="eventOutput = 'returnPress'"
+			@focus="eventOutput = 'focus'"
+			@blur="eventOutput = 'blur'" />
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="text:" horizontalAlignment="50%" />
+			<TextField v-model="text" id="TextField_text_textfield" horizontalAlignment="50%" />
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="hint:" />
+			<TextField v-model="hint" id="TextField_hint_textfield" />
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="editable:" />
+			<input type="checkbox" v-model="editable" id="TextField_editable_input">
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="maxLength:" />
+			<TextField v-model="maxLength" keyboardType="number" id="TextField_maxLength_input" />
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="secure:" />
+			<input type="checkbox" v-model="secure" id="TextField_secure_input">
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="keyboardType:" />
+			<select v-model="keyboardType" id="TextField_keyboardType_select">
+				<option value="datetime" id="TextField_keyboardType_dateTime_option">datetime</option>
+				<option value="phone" id="TextField_keyboardType_phone_option">phone</option>
+				<option value="number" id="TextField_keyboardType_number_option">number</option>
+				<option value="url" id="TextField_keyboardType_url_option">url</option>
+				<option value="email" id="TextField_keyboardType_email_option">email</option>
+			</select>
+		</StackLayout>
+		<StackLayout orientation="horizontal">
+			<Label text="returnKeyType:" />
+			<select v-model="returnKeyType" id="TextField_returnKeyType_select">
```

Comment:
can we test 'returnKeyType' prop ? as far as I remember, it was native only property to edit enter button on virtual keyboard

Reply:
I have added that prop's controls in case this test will be realized in a native app, then it shall come handy.